<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPort_Editer</class>
 <widget class="QDialog" name="SerialPort_Editer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="ConfirmBtn">
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="BaseTab">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string/>
             </property>
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>21</x>
                <y>20</y>
                <width>24</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>通道</string>
              </property>
             </widget>
             <widget class="QComboBox" name="ChannelCBX">
              <property name="geometry">
               <rect>
                <x>102</x>
                <y>21</y>
                <width>69</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>21</x>
                <y>60</y>
                <width>41</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>平台ID</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="SubPrjNameID">
              <property name="geometry">
               <rect>
                <x>101</x>
                <y>60</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLineEdit" name="SubPrjName">
              <property name="geometry">
               <rect>
                <x>101</x>
                <y>90</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_17">
              <property name="geometry">
               <rect>
                <x>21</x>
                <y>90</y>
                <width>41</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>点位名</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="SubPrjNum">
              <property name="geometry">
               <rect>
                <x>100</x>
                <y>120</y>
                <width>133</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_18">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>120</y>
                <width>51</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>分项编号</string>
              </property>
             </widget>
             <widget class="QLabel" name="Linklable">
              <property name="geometry">
               <rect>
                <x>260</x>
                <y>90</y>
                <width>91</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>获取平台参数</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>串口设置</string>
             </property>
             <widget class="QComboBox" name="ParityCBX">
              <property name="geometry">
               <rect>
                <x>252</x>
                <y>37</y>
                <width>69</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>198</x>
                <y>37</y>
                <width>48</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>奇偶校验</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_7">
              <property name="geometry">
               <rect>
                <x>21</x>
                <y>67</y>
                <width>36</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>数据位</string>
              </property>
             </widget>
             <widget class="QComboBox" name="BaudRateCBX">
              <property name="geometry">
               <rect>
                <x>63</x>
                <y>37</y>
                <width>69</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QComboBox" name="StopBitCBX">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>67</y>
                <width>69</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QComboBox" name="DataBitsCBX">
              <property name="geometry">
               <rect>
                <x>63</x>
                <y>67</y>
                <width>69</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_8">
              <property name="geometry">
               <rect>
                <x>21</x>
                <y>37</y>
                <width>36</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>波特率</string>
              </property>
             </widget>
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>198</x>
                <y>67</y>
                <width>36</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>停止位</string>
              </property>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>流量计</string>
             </property>
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>40</y>
                <width>41</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>厂家</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="Protocol">
              <property name="geometry">
               <rect>
                <x>310</x>
                <y>90</y>
                <width>90</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLineEdit" name="Manufacture">
              <property name="geometry">
               <rect>
                <x>450</x>
                <y>40</y>
                <width>90</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_20">
              <property name="geometry">
               <rect>
                <x>250</x>
                <y>90</y>
                <width>51</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>协议类型</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="SlaveID">
              <property name="geometry">
               <rect>
                <x>80</x>
                <y>40</y>
                <width>90</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_9">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>40</y>
                <width>51</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>节点地址</string>
              </property>
             </widget>
             <widget class="QLabel" name="Linklable_2">
              <property name="geometry">
               <rect>
                <x>560</x>
                <y>40</y>
                <width>91</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>获取平台参数</string>
              </property>
             </widget>
             <widget class="QLineEdit" name="Manufacture_Name">
              <property name="geometry">
               <rect>
                <x>300</x>
                <y>40</y>
                <width>90</width>
                <height>20</height>
               </rect>
              </property>
             </widget>
             <widget class="QLabel" name="label_10">
              <property name="geometry">
               <rect>
                <x>410</x>
                <y>40</y>
                <width>41</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>序列号</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="AlarmTab">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>报警开关</string>
             </property>
             <widget class="QCheckBox" name="EnableCBX">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>59</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>ENABLE</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
